prospero-cli/src/main/resources/UsageMessages.properties

```diff
@@ -17,13 +45,21 @@ prospero.repository.add.usage.header    = Add a maven repository to an installat
 prospero.repository.list.usage.header   = List maven repositories used by an installation.
 prospero.repository.remove.usage.header = Remove a maven repository from an installation.
 
-# Parameters
-usage.optionListHeading = %nOptions:%n
+#
+# Parameter Groups Headings
+#
+# By defining these, the parameter groups will be printed in separate sections introduced by these headings.
+
+localRepoOptions.heading = %nMaven cache:%n
+
+#
+# Option Descriptions
+#
 
 channel = Channel file URL.
 remote-repositories = URLs of remote Maven repositories that contains the artifacts required to install the application server ( multiple URLs are separated by comma).
 definition = Galleon provisioning XML definition file path.
-dir = Target directory where the application server is going to be provisioned.
+dir = Installation directory. If not specified, current working directory is used.
```

Comment:
> If not specified, current working directory is used.

Is it possible to have a different text for install command?

Reply:
Yes, I tried to make the text universal, but I think we can make different descriptions per command, if we want.

Reply:
I update this section so the --dir has specific description for the install command.